src/pages/[platform]/prev/build-a-backend/graphqlapi/query-data/index.mdx

```diff
@@ -45,7 +45,7 @@ Before you begin, you will need:
 
 <InlineFilter filters={["react-native"]}>
 
-- An [application connected to the API](/[platform]/prev/build-a-backend/graphqlapi/connect-to-api/)
+- An [application connected to the API](/[platform]/build-a-backend/graphqlapi/connect-to-api/)
```

Comment:
Looks like some lib-v1 GraphQL pages did not get moved over - will move these over then update prev/ doc links pointing to correct place as V5 users should be following the V5 set up and connect content